Fix skipper ingress binpacking setting

mikkeloscar opened the pull request:
Follow up to #5212 

The template function was used wrong and wasn't tested in e2e.

mikkeloscar committed:
Fix skipper ingress binpacking setting

Signed-off-by: Mikkel Oscar Lyderik Larsen <mikkel.larsen@zalando.de>

linki commented:
👍

szuecs commented:
@mikkeloscar this fails on cluster creation:

```
value: \"-618m\": must be greater than or equal to 0" module=skipper

time="2022-07-07T17:52:53Z" level=error msg="* spec.template.spec.containers[0].resources.requests[cpu]: Invalid value: \"-618m\": must be greater than or equal to 0" module=skipper

time="2022-07-07T17:52:53Z" level=info msg="statefulset.apps/skipper-ingress-redis unchanged" module=skipper

time="2022-07-07T17:52:53Z" level=error msg="The Deployment \"skipper-ingress\" is invalid: " module=skipper

time="2022-07-07T17:52:53Z" level=error msg="* spec.template.spec.containers[0].resources.limits[cpu]: Invalid value: \"-618m\": must be greater than or equal to 0" module=skipper

time="2022-07-07T17:52:53Z" level=error msg="* spec.template.spec.containers[0].resources.requests[cpu]: Invalid value: \"-618m\": must be greater than or equal to 0" module=skipper
```

mikkeloscar commented:
:+1:

mikkeloscar commented:
@szuecs This is now fixed, we had a low CPU setting (`100m`) for skipper-ingress in e2e which led to a negative number. Changed it to the default `1000m` now that we also run on dedicated node pool in e2e.

szuecs commented:
:+1:
